src/_includes/platforms/python/request-data.md

```diff
@@ -1 +1,3 @@
 Request data is attached to all events: **HTTP method, URL, headers, form data, JSON payloads**. Sentry excludes raw bodies and multipart file uploads. Sentry also excludes personally identifiable information (such as user ids, usernames, cookies, authorization headers, IP addresses) unless you set ``send_default_pii`` to ``True``.
+
+Scope data that is set from within a request handler/view or middleware will not be shared between requests. E.g. setting a tag during import time will set a global tag, while setting a tag within a view will attach the tag only to events sent as part of that request lifecycle.
```

Comment:
Maybe would be better to start like this: `Each request has a separated scope, this means...`